ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12.yaml

```diff
@@ -50,7 +50,9 @@ images:
 - dockerfile_literal: |
     FROM bin AS builder
     FROM registry.ci.openshift.org/ci/tests-private-base:4.12
-    RUN mkdir -p /go/src/github.com/openshift
+    RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm && \
```

Comment:
@memodi what you add is tests-private-newpr, not tests-private-ui. please correct it.